scripts/release/util/check_sig.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# shellcheck disable=2045
+
+if [ $# -eq 0 ]
```

Comment:
you want to check that you have a single argument, rather then checking that the user forgot to specify a directory.

Reply:
Will change.